x/auth/client/tx.go

```diff
@@ -29,7 +29,7 @@ import (
 //
 // TODO:/XXX: Using a package-level global isn't ideal and we should consider
 // refactoring the module manager to allow passing in the correct module codec.
-var Codec authtypes.Codec
```

Comment:
+++++ This was causing me a headache on ethermint

Reply:
Sorry, I didn't get this. 
Codec is of type codec.Marshaler now, authtypes.Codec interface has been eliminated.

Reply:
yeah, I meant that I was glad this was changed 🙂

Reply:
Seems like validation that switching to `Any` was the right move 👍

Reply:
yeah I had to manually override the auth client codec to the Ethermint's one because the Ethermint Account type wasn't registered